### PR TITLE
made some working server and client which can interact

### PR DESCRIPTION
**Pull request by tetie12345**
i made 2 scripts which can interact with eachother, while janky, slightly functional. see video below for usage and functionality

https://github.com/user-attachments/assets/4aa4f3fe-86d2-4327-bf30-c45ce5634e34

### CODE REVIEW

File: client.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import asyncio
+from websockets.asyncio.client import connect
```
Comment (tetie12345) [inline client.py:2]: websockets.asyncio doesn't work on Linux, will find an alternative later

File: server.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import asyncio
+from websockets.asyncio.server import serve
```
Comment (tetie12345) [inline server.py:2]: websockets.asyncio doesn't work on Linux, will find an alternative later

**Review by tetie12345 — COMMENTED** [review]
Seems to work in most cases, but quite janky without inyerrupts

File: server.py (hunk 2 above)
Comment (noahniemeijer) [reply server.py:2]: Can we use both? like that we use the library os for example to determine which operating system the client is using. Or will that not work for cross-platform?

**Comment by noahniemeijer** [comment]
Looks good! I will merge it

**Pull request merged**